src/main/java/RunConfig.java

```diff
@@ -12,7 +12,19 @@ public static String readFromConfig(String stringProp) throws IOException {
         InputStream input = new FileInputStream("src/main/resources/system.properties");
         prop.load(input);
         return prop.getProperty(stringProp);
+    }
+
+    public static String getStartURL() {
 
+        try {
+            Properties prop = new Properties();
+            InputStream input = new FileInputStream("src/main/resources/system.properties");
+            prop.load(input);
+            return prop.getProperty("basicURL");
+        }  catch (IOException e) {
+            e.printStackTrace();
+            return null;
```

Comment:
return null = potential for NullPointerException. are you sure all the places where you use it will be ready for null? maybe "" is better? does it even make sense to continue if we were unable to read start url from config? maybe let's just fail?